src/views/Give/CausesDashboard.tsx

```diff
@@ -69,11 +69,16 @@ export default function CausesDashboard() {
         }),
       );
     } else {
+      // Since the new version of the YieldDirector contract uses IDs as a deposit indexing system
+      // the async thunk used to interact with the contract needs to have an ID passed to it
+      // here we use the ID of "-1" to indicate to the thunk that this is a new deposit and thus
```

Comment:
Can we extract `-1` into a constant with a meaningful name and use that across the codebase? That also reduces the number of comments that you need to litter everywhere.